item/historical/scripts/T002.py

```diff
@@ -0,0 +1,112 @@
+"""Data cleaning code and configuration for T002."""
+from .util.managers.dataframe import ColumnName
+from functools import lru_cache
+import pandas as pd
+
+#: Dimensions and attributes which do not vary across this data set.
+COMMON_DIMS = dict(
+    # Add the same source to all rows since all data comes from the same source
+    source="International Transport Forum",
+    # Since all the data is associated to "Freight," the Service is "Freight"
+    service="Freight",
+    # The dataset does not provide any data on the following columns, so we
+    # add the default value of "All" in both cases
+    technology="All",
+    fuel="All",
+    vehicle_type="Container",
+)
+
+#: Columns to drop from the raw data.
+COLUMNS = dict(
+    drop=[
+        "COUNTRY",
+        "VARIABLE",
+        "YEAR",
+        "Unit Code",
+        "PowerCode Code",
+        "PowerCode",
+        "Reference Period Code",
+        "Reference Period",
+        "Flag Codes",
+        "Flags",
+    ],
+    # Column containing country name for determining ISO 3166 alpha-3 codes and
+    # iTEM regions. Commented, because this is the default value.
+    # country_name='Country',
+)
+
+
+def process(df):
+    """Process data set T002."""
+    # Getting a generic idea of what countries are missing values and dropping
+    # NaN values
+    #
+    # Rule: Erase all value with NaN
+
+    list_of_countries_with_missing_values = list(
```

Comment:
Note that because these lines were repeated in several places, we now have a function that does the same thing. See e.g. https://github.com/transportenergy/database/blob/01ec2faaa808194b088d9b23d64d26aff5c1f99b/item/historical/scripts/T000.py#L54-L55

Please use this instead.